scripts/build/linting.mk

```diff
@@ -31,20 +31,20 @@ golangci-fix:
 
 golines:
 	@echo "--> Running golines"
-	@./build/scripts/golines.sh
+	@./scripts/build/golines.sh
 
 #################
 #    license    #
 #################
 
 license:
 	@echo "--> Running addlicense with -check"
-	(go run github.com/google/addlicense -check -v -f $(ROOT_DIR)/LICENSE.header beacond build examples kurtosis mod testing) || exit 1;
+	(go run github.com/google/addlicense -check -v -f $(ROOT_DIR)/LICENSE.header -ignore "contracts/**" .) || exit 1;
```

Comment:
why we don't ignore anymore kyrtosis and beacond?

Reply:
I understand we need to ignore contract because there is no mod folder anymore?

Reply:
I changed this line to run the addlicense check on all fodlers except the contracts folder, instead of passing it the folder it needs to check. So before kurtosis and beacond werent ignored